packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/service/NotificationSchedulingHelper.java

```diff
@@ -29,11 +29,13 @@
 import expo.modules.notifications.notifications.model.Notification;
 import expo.modules.notifications.notifications.model.NotificationRequest;
 
+import static android.content.Context.ALARM_SERVICE;
+
 /**
  * {@link JobIntentService} responsible for handling events related to scheduled notifications:
  * fetching, adding, removing and triggering. Work should be enqueued with #enqueueVerb static methods.
  */
```

Comment:
`NotificationSchedulingHelper` no longer `extends JobIntentService` so we should update this comment

Reply:
and remove the `JobIntentService` import